src/components/pricedButton/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Button from 'react-toolbox/lib/button';
+import { fromRawLsk } from '../../utils/lsk';
+import styles from './pricedButton.css';
+
+const PricedButton = (props) => {
```

Comment:
A pattern I like is to destructure props out here for less verbosity in the component body.